lib/cogs/random.py

```diff
@@ -29,6 +30,14 @@ async def coinflip(self, ctx):
         else:
             embed = Embed(title='TAILS', color=0xca9502)
         await ctx.send(embed=embed)
+        
+    @command(aliases=['flag'])
```

Comment:
This will conflict with the function name.

```suggestion
    @command()
```